libcontainer/cgroups/fs2/hugetlb.go

```diff
@@ -27,10 +26,7 @@ func setHugeTlb(dirPath string, r *configs.Resources) error {
 }
 
 func statHugeTlb(dirPath string, stats *cgroups.Stats) error {
-	hugePageSizes, err := cgroups.GetHugePageSize()
-	if err != nil {
-		return fmt.Errorf("failed to fetch hugetlb info: %w", err)
-	}
+	hugePageSizes, _ := cgroups.GetHugePageSize()
```

Comment:
Could you add a comment to clarify that we are ignoring err on purpose?

Reply:
This is kind of an intermediate commit, merely fixing the issue. Please see #3234 for a complete solution -- in where I remove the returned error altogether (as no one is using it), so there's no need to ignore it (and explain why).